src/bucket.js

```diff
@@ -71,6 +81,7 @@ export default class Bucket {
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
+  @nobatch("Batch doesn't support read operations.")
```

Comment:
I see... I'm super worried people will try to consume the results within the batch function, though that's probably matter of documentation. Hope you'll help writing these ;)